application/dialogs/AccountSelectDialog.cpp

```diff
@@ -1,85 +0,0 @@
-/* Copyright 2013-2015 MultiMC Contributors
- *
```

Comment:
It was replaced by AccountsDialog. The accounts page and AccountSelectDialog where in functionality and look almost identical. Now there is instead an AccountsWidget that is used by both the dialog and the page.